README.md

```diff
@@ -7,3 +7,30 @@ For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/
 --- 
 
 This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+
+
+(1) Added the .env.test and .env.development files.
+(2) npm install and npm install express.
+(3) created the controller and model files.
+
+create and seed database
+(1) type 'sudo service postgresql start' in the terminal
+(2) type 'npm run setup-dbs' in the terminal to create the database
+(3) type 'npm run seed' in the terminal to insert values into the tables
+
+
+Test EndPoint code
+(1) I made the get app code then exported the property to the controller. 
```

Comment:
It's really great that you've documented the process, but this normally would be in a README file! There's a useful resource [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes) for what a README normally contains